src/Data/CurrentUser/fetchWhoAmIWithToken.ts

```diff
@@ -16,29 +21,22 @@ export async function fetchWhoAmIWithToken(): Promise<WhoAmI | null> {
   const { url, headers: baseHeaders } = whoAmIConfig
 
   const headers = { ...baseHeaders, Authorization: tokenAuthorization }
+  const client = createRestApiClient(url, { headers })
 
-  // TODO: Replace fetch with pisaClient, once #11616 is resolved
-  let response: Response
   try {
-    response = await fetch(url, { method: 'GET', headers })
+    const response = await client.get('')
+    return response as WhoAmI
```

Comment:
* [ ] FYI instead of `await` we could return the promise directly. `return client.get('') as Promise<WhoAmI>`

Reply:
No 😸. Because this `await` is important for the `try/catch` block.